### PR TITLE
WK2-878 Performance guide

### PR DESCRIPTION
**Pull request by tom-meader-wk**
# Checklist

- I have spellchecked my changes.
- I have updated the API Documentation CHANGELOG where applicable.
- I have added my guides to Welcome.md where applicable
- I have added my guides to toc.json where applicable.

### CODE REVIEW

File: docs/performance/Basics.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+<!-- theme: warning -->
+> If WealthKernel does not know how your fees are accrued, these figures will not be accurate.
+
+# How does our performance calculation work?
+
+For a given day we calculate the rate of return as 
+
+<!--TODO: Could make this mermaid diagram when stoplight supports mermaid v10.9.0 https://mermaid.js.org/config/math.html -->
+
+![grossperf-3.png](../../assets/images/grossperf-3.png)
```
[the remark this answers is not in the document]
Comment (tom-meader-wk) [reply docs/performance/Basics.md:20]: That was the first thing I tried, sadly not. Then I saw mermaid does, but we're on too old a version 😒

File: docs/performance/Basics.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+or
+
+![netperf-3.png](../../assets/images/netperf-3.png)
+
+In other words, we take the value at the end of the current interval, add on expected income, subtract cashflow, (and accrued fees for the net figure) and divide by the end value at the end of the previous interval.
```
Comment (niall-maloney) [inline docs/performance/Basics.md:26]:
```suggestion
We take the value at the end of the current interval, add the expected income, subtract the cashflow, for the net figure any accrued fees are also subtracted. We then divide this by the end value from the end of the previous interval.
```

File: docs/performance/Basics.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## Simple
+
+In the most simple example, there are no fees, dividends or cash flow. For a value of £100, measuring the TWRR for an increase in value to £102 the next day gives a value of 0.02 or 2%.
+
+|Day|StartValue|EndValue|Performance Gross|Performance Net|
+|--------|--------|--------|--------|--------|
+|Day n|£0|£100|-|-|
+|Day n+1|£100|£100|((100+0-0)/100)-1=0 or 0%|((100+0-0)/100)-1=0 or 0%|
+|Day n+2|£100|£102|((102+0-0)/100)-1=0.02 or 2%|((102+0-0-0)/100)-1=0.02 or 2%|
```
Comment (niall-maloney) [inline docs/performance/Basics.md:43]: We should add some spacing to the calculations to make them easier to read.

File: docs/performance/Basics.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+---
+tags: [Performance]
+---
+
+# Performance basics
+
+There are many different methods to calculate performance. We offer an API for one such method, Time Weight Rate of Return (TWRR), as the effect of cash flow can be filtered out. We calculate two figures using this methodology, one gross figure and one net of fees.
```
Comment (niall-maloney) [inline docs/performance/Basics.md:7]:
```suggestion
WealthKernel calculates performance using the Time-Weighted Rate of Return (TWRR) methodology, as this provides a better view of performance for investors by filtering out the effect of cash flow. We calculate two figures using this methodology: a gross figure and a net-of-fees figure.
```

**Review by niall-maloney — COMMENTED** [review]
Should we add some FAQs? e.g. Why did performance drop? Because you're not telling us how fees are accrued.

**Review by niall-maloney — APPROVED** [review]

**Pull request merged**